### PR TITLE
minor netdata improvements

### PR DESCRIPTION
**Pull request by ktsaou**
1. converted netdata charts to `area` charts

    I think charts like this:

     image

    are nicer, than this:

    image

2. added netdata to `trace_flag` output

3. fixed compiler warning about `setuid()` return value being ignored, by calling `perror()` on `setuid()` errors.

4. fix 100% cpu consumption introduced with https://github.com/schweikert/fping/commit/7c7e007502a3a2bd435a9e6b797387dbea8b992c

### CODE REVIEW

File: src/fping.c [hunk 1]
```diff
@@ -1099,6 +1101,8 @@ void main_loop()
         /* Make sure we don't wait too long, in case a report is expected */
         if( report_interval && ( loop_flag || count_flag ) ) {
                wait_time_next_report = timeval_diff ( &current_time, &next_report_time );
+               if(wait_time_next_report < 0)
+                    wait_time_next_report = -wait_time_next_report;
```
Comment (schweikert) [inline src/fping.c]: This doesn't look right. Why did you add this?

Comment (schweikert) [inline src/fping.c]: I have added a commit that should fix this.

Comment (ktsaou) [inline src/fping.c]: ok, removed it.

**Comment by ktsaou** [comment]
perfect! thanks!
